#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raises a NameError exception with a message and exit"""
    raise NameError(message)
